packages/atomic-layout/src/utils/getCompatibleMatchMedia.ts

```diff
@@ -0,0 +1,19 @@
+import matchMediaMock from 'match-media-mock'
+import { Layout, isServer } from '@atomic-layout/core'
+
+let matchMediaOnServer: any
+
+if (isServer()) {
+  matchMediaOnServer = matchMediaMock.create()
```

Comment:
The purpose of `matchMediaMock` is to emulate native `window.matchMedia` functionality in node environment. That implies that you can actually match screen size towards media query definition. In other words, it brings the entire `matchMedia` functionality to a non-browser environment.

For our purpose it's too expensive to take that functionality mock if we only need to check a) if we are on a server, b) if a prop breakpoint name is the default breakpoint. I'd suggest you remove this differentiation `getCompatibleMatchMedia`.